### DISCUSSION TITLE
pause or stop - any pros/cons?

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#401] [Q&A] [answered]
Since there are times like overnight I don't need the bridge running, is there any benefits or negative between pause/unpause or stop/start? Figuring I can just create tasks or shortcuts to pause/unpause or stop/start as I need?  Seems like an easy way to limit unnecessary data usage since I have not been able to get LAN mode to work and no one seems to have any suggestions.

**Comment by mrlt8** [comment, marked as answer]
I think that should work fine.

Hoping to get on-demand streaming for each camera working in v2.0.

**Reply by cheme75** [reply]
If I could figure out the lan issue, it would be a big help. Meanwhile, manually running pause or stop doesn’t seem to be much different.

**Reply by mrlt8** [reply]
Could you try running the bridge on a different machine or in a Linux VM?

**Reply by cheme75** [reply]
Only other is a win 10 home laptop - can’t see another win machine doing any diff. Thought about just converting it to Linux but the other container connects with a desktop app that’s not available on Linux yet.  I’ve looked all over the firewall but nothing stands out to me, but I’m not exactly fluent in knowing what might be blocked that should be open.

**Reply by mrlt8** [reply]
I'm not very familiar with windows 10, but maybe the built-in firewall is preventing local UDP traffic from the bridge?

**Reply by cheme75** [reply]
> I'm not very familiar with windows 10, but maybe the built-in firewall is preventing local UDP traffic from the bridge?

Yes, that could be it. Only things I’ve had to add were tcp rules to open the 3 specified ports. If cams to/from bridge lan traffic uses udp, then that could be it. Are there specific protocols and/or ports?

**Reply by maxfield-allison** [reply]
honestly you could safely disable the windows firewall on that box at least to test. if someone's in your network past your edge, you're already in trouble lol.

**Reply by cheme75** [reply]
I opened all ports to udp traffic on the private network and that did not work. So, I considered disabling the firewall, at least for testing - can’t hurt to try. I just feel the win10 firewall is more to keep something that might happen to get on this machine via poor judgement like a risky download from getting out beyond this machine, so opening a port for a specific app isn’t too risky compared to no firewall at all. But then the other machines firewalls should block any bad actors, so probably not a huge risk as long as I continue to be careful with my downloads.
